x-pack/plugins/security_solution/public/common/components/navigation/breadcrumbs/index.ts

```diff
@@ -68,9 +72,11 @@ const isAdminRoutes = (spyState: RouteSpyState): spyState is AdministrationRoute
 // eslint-disable-next-line complexity
 export const getBreadcrumbsForRoute = (
   object: RouteSpyState & TabNavigationProps,
-  getUrlForApp: GetUrlForApp
+  getUrlForApp: GetUrlForApp,
+  basePath: IBasePath
 ): ChromeBreadcrumb[] | null => {
   const spyState: RouteSpyState = omit('navTabs', object);
+  const siemRootBreadcrumb = fullSiemRootBreadcrumb(basePath);
```

Comment:
Not sure if it's worth it but do we need to add tests for this specific functionality? Like a test using a base path and checking that the SIEM URL includes it?